tasks/dataset-split/src/imageDatasetSplit.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from coretex import currentTaskRun, NetworkDataset, ImageDataset, ImageSample
+
+from .utils import splitOriginalSamples
+
+
+def imageDatasetSplit(originalDataset: ImageDataset, datasetParts: int, projectId: int) -> list[NetworkDataset]:
+    splitSamples: list[list[ImageSample]] = splitOriginalSamples(originalDataset.samples, datasetParts)
+    
+    splitDatasets: list[NetworkDataset] = []
+
+    for index, sampleChunk in enumerate(splitSamples):
+        splitDataset = ImageDataset.createDataset(f"{currentTaskRun().id}-split-dataset-{index}", projectId)
+        splitDataset.saveClasses(originalDataset.classes)
```

Comment:
What happens here in case dataset you will be splitting has 3 classes and after split on three parts each of the parts has images of only one class? If I am not missing something than you will have three datasets and all three with 3 classes but from those 3 classes your images are from only 1 class for each of the datasets. Correct me if I'm wrong on this one.